po/radxa-welcome.pot

```diff
@@ -1,27 +1,27 @@
-# Translations for Radxa's welcome app
-# Copyright (C) 2024 Radxa Computer Co., Ltd
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
 # This file is distributed under the same license as the radxa-welcome package.
-# Radxa Computer Co., Ltd <dev@radxa.com>, 2024.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```

Comment:
They became placeholder values again.

Reply:
I have no idea how to fix that, it seems it isnt mentioned on meson docs